@here/harp-mapview/lib/DataSource.ts

```diff
@@ -292,27 +365,87 @@ export abstract class DataSource extends THREE.EventDispatcher {
     }
 
     /**
-     * The minimum zoom level to use for display.
-     *
-     * @returns The minimum zoom level to use for display.
+     * The minimum zoom level at which data is available or displayed at
+     * (depending on [[DataSource]] subclass).
+     * @deprecated Use [[minDataLevel]] and [[minDisplayLevel]] instead.
      */
     get minZoomLevel(): number {
-        return this.m_minZoomLevel;
+        logger.warn(
+            "DataSource.minZoomLevel is deprecated. Use minDataLevel and maxDataLevel instead."
+        );
+        return this.m_minDataLevel;
     }
 
     set minZoomLevel(level: number) {
-        this.m_minZoomLevel = level;
+        logger.warn(
+            "DataSource.minZoomLevel is deprecated. Use minDataLevel and minDisplayLevel instead."
+        );
+        this.m_minDataLevel = level;
+        this.m_minDisplayLevel = level;
     }
 
     /**
-     * The maximum zoom level to use for display.
+     * The maximum zoom level at which data is available or displayed at
+     * (depending on [[DataSource]] subclass).
+     * @deprecated Use [[maxDataLevel]] and [[maxDisplayLevel]] instead.
      */
     get maxZoomLevel(): number {
-        return this.m_maxZoomLevel;
+        logger.warn(
+            "DataSource.maxZoomLevel is deprecated. Use maxDataLevel and maxDisplayLevel instead."
+        );
+        return this.m_maxDataLevel;
     }
 
     set maxZoomLevel(level: number) {
-        this.m_maxZoomLevel = level;
+        logger.warn(
+            "DataSource.maxZoomLevel is deprecated. Use maxDataLevel and maxDisplayLevel instead."
+        );
+        this.m_maxDataLevel = level;
+        this.m_maxDisplayLevel = level;
```

Comment:
this won't work for OLP, they have a maxZoomLevel of 1, but they want to display it up to level 9

Reply:
Nevermind, just read this is deprecated.